language-server/src/server.js

```diff
@@ -72,6 +78,9 @@ connection.onInitialize(({ capabilities, workspaceFolders }) => {
 });
 
 connection.onInitialized(async () => {
+  if (hasConfigurationCapability) {
+    connection.client.register(DidChangeConfigurationNotification.type, undefined);
```

Comment:
It's odd to explicitly pass `undefined` to a function. Unless you have a good reason, let's remove that parameter.

Reply:
I removed it and tested, works without this parameter too. My bad, I took a reference from how it's done in vscode lsp extension.